ush/ufoeval/run_ufo_hofx_test.sh

```diff
@@ -71,7 +71,7 @@ machine=${machine:-orion}
 
 if [ $machine = orion ]; then
    if [ $run_filtering == NO ]; then
-      workdir=/work2/noaa/da/$LOGNAME/ufoeval/$cycle/${obtype}_noqc
+      workdir=/work2/noaa/da/$LOGNAME/ufoeval_conv_ps/$cycle/${obtype}_noqc
```

Comment:
Will this path work for all ob types or just Ps?

Reply:
Ha!  Fixed.  Thanks.